app/controllers/cases_controller.rb

```diff
@@ -4,4 +4,20 @@ def index
     @p_case = @patient.cases.build
     @urgent_cases = Case.where(urgent_flag: true)
   end
+
+  def search
+    name_match = Patient.where(name: params[:search])
+    primary_match = Patient.where(primary_phone: params[:search])
+    secondary_match = Patient.where(secondary_phone: params[:search])
+    patients = name_match | primary_match | secondary_match
```

Comment:
Had no idea that `|` was a concatenation operator for arrays! Learn something new every day. Thanks for showing me this!